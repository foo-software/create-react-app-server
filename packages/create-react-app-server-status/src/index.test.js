import * as packageExports from '.';

describe('@foo-software/create-react-app-server-status', () => {
  it('should match snapshot', () => {
    expect(packageExports).toMatchSnapshot();
  });
});
