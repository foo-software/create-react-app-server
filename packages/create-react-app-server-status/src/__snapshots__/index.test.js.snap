// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@foo-software/create-react-app-server-status should match snapshot 1`] = `
Object {
  "CreateReactAppServerStatus": [Function],
}
`;
